### PR TITLE
Binary-Search and BubbleSort added by architkshk

### PR DESCRIPTION
**Pull request by architkshk**
Description:
In this Pull Request I've added Binary Search and Bubble Sort program in C.
#1 #25 #27

### CODE REVIEW

File: Binary_search/archit_binarysearch.c [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
```
Comment (MJ10) [inline Binary_search/archit_binarysearch.c]: Please move this file to the `BinarySearch` directory and rename it to your github username.

File: bubblesort/architkshk_bubblesort.c [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
```
Comment (MJ10) [inline bubblesort/architkshk_bubblesort.c]: Please move this file to the `BubbleSort` directory and rename it to your github username.

**Comment by MJ10** [comment]
@architkshk Any updates?

**Comment by salman-bhai** [comment]
Pull Request will be closed in 10 hours. Please send required changes before that.

**Comment by architkshk** [comment]
Sorry for the delay. I've committed the change. Please review.

**Pull request merged**